### PR TITLE
feed: Add VirusTotal Retrohunt feed

### PR DESCRIPTION
**Pull request by Diviei**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Add VirusTotal Livehunt as a Cortex feed to the pack.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: .hooks/pre-commit [hunk 1]
```diff
@@ -78,4 +78,4 @@ if [ -z "$1"  ]; then
     fi
 fi
 
-echo ""
```
Comment (bziser) [inline .hooks/pre-commit:81]: please revert this change

**Comment by bziser** [comment]
@Diviei thanks for the contribution! Could you fix these [errors](https://app.circleci.com/pipelines/github/demisto/content/153394/workflows/c3beee03-a995-4376-8812-37e6f439e731/jobs/498650)?

**Comment by bziser** [comment]
Please these [errors](https://app.circleci.com/pipelines/github/demisto/content/153773/workflows/059f035e-bfe7-4dfd-83f8-34cf6d15f2ed/jobs/499289)

**Comment by Diviei** [comment]
> Please see these [errors](https://app.circleci.com/pipelines/github/demisto/content/153773/workflows/059f035e-bfe7-4dfd-83f8-34cf6d15f2ed/jobs/499289)

@bziser Looks like circleci didn't run for the last commit. Can you take a look?

**Review by bziser — APPROVED** [review]
